CHANGELOG.md

```diff
@@ -30,6 +30,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* Sources priority in Podfile warks as stated in documentation.
```

Comment:
nit: We've got a typo -- 'warks' should be 'works'.

Reply:
Thanks! Fixed